docker-compose.yml

```diff
@@ -1,7 +1,8 @@
 version: "3.9"
+
 services:
   db:
-    image: postgres:10.9
+    image: postgres:10.9-alpine
```

Comment:
this `alpine` image works the same but is smaller, easier to download and start working.
 
![image](https://user-images.githubusercontent.com/7331511/175760733-0a5a464b-6e0d-419e-adb9-aceafdb558f8.png)